src/poke/mod.rs

```diff
@@ -49,7 +49,7 @@ pub fn load_session_settings(
         };
         if name == "deployer" {
             initial_deployer = Some(account.clone());
-            deployer_address = Some(account.address.clone());
+            default_deployer_address = Some(account.address.clone());
```

Comment:
I'm confused a little bit... It is a general comment or something I should work on with regards to line 52 in `src/poke/mod.rs` ?

Reply:
It's a feedback on the approach, yes :)
Instead of 
```toml
[contracts.sip-010-trait-ft-standard]
path = "contracts/external/sip-10-ft-standard.clar"
deployer = "SP3FBR2AGK5H9QBDH3EEN6DF8EK8JY7RX8QJ5SVTE"
depends_on = []
```

introducing a level of indirection based on the labels
```toml
[contracts.sip-010-trait-ft-standard]
path = "contracts/external/sip-10-ft-standard.clar"
deployer = "wallet-1"
depends_on = []
``` 

would let us pick different wallets dynamically, based on the network Devnet/Testnet/Mainnet that you're working with.
So if you're deploying on Mainnet, clarinet, when loading `settings/Mainnet.toml`, should be retrieving all the deployers for all your contracts, and use the adequate private keys to sign the contract publish transactions.